pyaedt/edb.py

```diff
@@ -260,6 +262,19 @@ def cell_names(self):
             names.append(cell.GetName())
         return names
 
+    @property
+    def excitations(self):
+        """Retrieve the Terminals of the active edb.
+
+        Returns
+        -------
+        dict
+        """
+        terminals = {}
+        for terminal in list(self.active_layout.Terminals):
+            terminals[terminal.GetName()] = Terminal(self, terminal)
```

Comment:
A excitation consises of two terminals, so if we count the number of exicitation by couting terminals reference terminals should be excluded.